kpi/renderers.py

```diff
@@ -49,4 +59,4 @@ class XlsRenderer(renderers.BaseRenderer):
     def render(self, data, media_type=None, renderer_context=None):
         asset = renderer_context['view'].get_object()
         return asset.to_xls_io(versioned=self.versioned,
-                               kobo_specific_types=self.kobo_specific_types)
+                               kobo_specific_types=self.kobo_specific_types)
```

Comment:
Our editors love to fight each other about this.
https://www.reddit.com/r/Python/comments/1zjugg/question_why_does_pep8_recommend_leaving_a_blank/

Reply:
The problem is not my IDE, it's me. I know about the blank line at the end (and `PyCharms` too) but sometimes I forget.